$NetBSD: patch-apr-config.in,v 1.4 2023/04/07 14:57:58 wiz Exp $

* Do not detect pkgsrc buildlink3 environment as crosscompile.
  Fix consumers' builds.

--- apr-config.in.orig	2023-03-13 10:29:04.000000000 +0000
+++ apr-config.in
@@ -64,7 +64,7 @@ if test "$cross_compiling" != "no"; then
     # absolute path, but not installed path - we're cross compiling
     case "$normalized_command" in
       "${normalized_bindir}/"*) ;;
-      "/"*)                     location=crosscompile;
+      "/"*)                     #location=crosscompile;
                                 APR_TARGET_DIR=${normalized_command%${normalized_bindir}/apr-${APR_MAJOR_VERSION}-config};
                                 ;;
       *)                        ;;
